contentcuration/contentcuration/frontend/channelEdit/components/QuickEditModal/__tests__/EditBooleanMapModal.spec.js

```diff
@@ -182,26 +179,6 @@ describe('EditBooleanMapModal', () => {
         expect(dailyLifeValue).toBe(CheckboxValue.CHECKED);
         expect(foundationsValue).toBe(CheckboxValue.CHECKED);
       });
-
-      test('checkbox option should be indeterminate if not all nodes have the same options set', () => {
-        nodes['node1'].categories = {
-          [Categories.DAILY_LIFE]: true,
-          [Categories.FOUNDATIONS]: true,
-        };
-        nodes['node2'].categories = {
-          [Categories.DAILY_LIFE]: true,
-        };
-
-        const wrapper = makeWrapper({ nodeIds: ['node1', 'node2'] });
-
-        const optionsValues = getOptionsValues(wrapper);
-        const {
-          [Categories.DAILY_LIFE]: dailyLifeValue,
-          [Categories.FOUNDATIONS]: foundationsValue,
-        } = optionsValues;
-        expect(dailyLifeValue).toBe(CheckboxValue.CHECKED);
-        expect(foundationsValue).toBe(CheckboxValue.INDETERMINATE);
-      });
```

Comment:
Thanks for cleaning up the tests, too! I hadn't remembered about that, but it's great to stay on top of that 🎉